zephyr/Kconfig

```diff
@@ -0,0 +1,680 @@
+# Copyright (c) 2017-2020 Linaro Limited
+# Copyright (c) 2020 Arm Limited
+# Copyright (c) 2022 Legrand North America, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Option to build the project with the MCUBoot application
+config MCUBOOT
+	default n
+	bool  "MCUBoot interfaces"
+	select MCUBOOT_APP
+
+config MCUBOOT_APP
+	default n
+	bool  "MCUBoot application"
+	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
+	select USE_DT_CODE_PARTITION if HAS_FLASH_LOAD_OFFSET
+	select MCUBOOT_BOOTUTIL_LIB
+
+config MCUBOOT_BOOTUTIL_LIB_OWN_LOG
+	default n if MCUBOOT_APP
+
+config BOOT_USE_MBEDTLS
+	bool
+	# Hidden option
+	default n
```

Comment:
`default n` is not needed

Reply:
It pre-existed in the moved file.  Since this was a "refactor" I wanted to minimize the number of unnecessary changes.  Cleaning out 'default n' is not a necessary change for this "refactor".

Reply:
This pre-existed in the moved file. In order to reduce conflict and minimize the size of the PR, I chose to only make necessary changes.  Cleaning out unnecessary `default n` would simply add noise to this PR.